openapi.yml

```diff
@@ -0,0 +1,578 @@
+openapi: 3.0.0
```

Comment:
wondering if there's any way to indicate with openapi that an access token is required ...

Reply:
@ndushay There is! It's at the bottom of the file: https://github.com/sul-dlss/preservation_catalog/pull/1389/files#diff-dc51a340f5f12b9af5a5462818e6eca1R572-R578 Resolving.